web-common/src/lib/create-throttler.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Run callback once every `time`ms
+ */
+export function createThrottler(time: number) {
```

Comment:
Might be better if this function also accepted the `callback` as a parameter, so the component code could be a one-liner like:
```
throttle(myfunc(), 500)
```

Reply:
Check the usage in `ColumnProfile.svelte`, The entire call is within a reactive statement. If we had the callback during instantiation we would need to manually add all dependants in the reactive statement. The way it is right now, it is simpler to use in a component.